lms/djangoapps/courseware/tabs.py

```diff
@@ -35,7 +35,6 @@ class CoursewareTab(EnrolledTab):
     """
     type = 'courseware'
     title = ugettext_noop('Course')
-    priority = 10
```

Comment:
@Arti3DPlayer As per the [official docs](https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_assets/pages.html#reorder-the-pages), the position of "Course" and "Home" tabs should not be changed (i.e. they should always be in the first two positions). So I think we should leave their respective priorities as is, and only remove the others.